pokeapp/src/styling/components/_pokemon-card.scss

```diff
@@ -13,6 +13,117 @@
         justify-content: space-between;
         align-items: baseline;
     }
+
```

Comment:
We can get rid of the background color, border radius, and padding by using this syntax. `&-fire, &-water, etc`. The only unique value for each selector would be the box shadow color.